packages/node-opcua-data-model/source/diagnostic_info.ts

```diff
@@ -140,6 +140,19 @@ export class DiagnosticInfo extends BaseUAObject {
     public decodeDebug(stream: BinaryStream, options: DecodeDebugOptions): void {
         decodeDebug_DiagnosticInfo(this, stream, options);
     }
+
+    public static filterForResponse(diagnostic: DiagnosticInfo, mask: number, enumeration: DiagnosticInfo_Mask): DiagnosticInfo {
```

Comment:
I would rename `enumeration` to `diagnosticInfoMask`
and `mask` to `requestedDiagnostics`.